lua/octo/writers.lua

```diff
@@ -370,6 +370,7 @@ function M.write_comment(bufnr, comment, kind, line)
   end
   local comment_vt_ns = api.nvim_create_namespace("")
   M.write_virtual_text(bufnr, comment_vt_ns, line - 1, header_vt)
+  local header_mark = api.nvim_buf_set_extmark(bufnr, constants.OCTO_HEADER_NS, line - 1, 0, { end_line = line })
```

Comment:
We should only need to check the `comment.viewerDidAuthor` and change the way we write the header

Reply:
Closing in regards of [this comment](https://github.com/pwntester/octo.nvim/pull/116#issuecomment-789578727).